sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/models/SearchOptions.java

```diff
@@ -370,8 +373,8 @@ public SearchOptions setMinimumCoverage(Double minimumCoverage) {
      * @param orderBy the orderBy value to set.
      * @return the SearchOptions object itself.
      */
-    public SearchOptions setOrderBy(List<String> orderBy) {
-        this.orderBy = orderBy;
+    public SearchOptions setOrderBy(String... orderBy) {
+        this.orderBy = (orderBy == null) ? null : java.util.Arrays.asList(orderBy);
```

Comment:
nit:  `java.util.Arrays.asList` could be imported

Reply:
This is a code customization change where auto-import isn't able to easily resolve this, used a fully-qualified call while more functionality is added to the customization framework.